subprojects/packagefiles/gee/meson.build

```diff
@@ -2,7 +2,9 @@ project(
   'libgee',
   'c',
   'vala',
-  version: '0.20.6',
+  version: '0.20.8',
+  meson_version: '>=0.54.0',
```

Comment:
I assume this is used for override dependency, which is handled alternatively as

```
if meson.version().version_compare('>=0.54.0')
```

I wonder if we should mandate a minimum meson_version. 0.54 is quite old.